ietf-layer0-types.yang

```diff
@@ -1293,14 +1285,49 @@ module ietf-layer0-types {
         "Maximum acceptable accumulated chromatic dispersion
          on the receiver";
     }
-    list chromatic-and-polarization-dispersion-penalty {
+    list chromatic-dispersion-penalty {
       config false;
       description
         "Optional penalty associated with a given accumulated
          CD and PMD.
          This list of triplet cd, pmd, penalty can be used to
          sample the function penalty = f(CD, PMD).";
-      uses cd-pmd-penalty ;
+      leaf chromatic-dispersion {
+        type union {
+          type decimal64 {
+            fraction-digits 2;
+            range "0..max";
+          }
+          type empty;
+        }
+        units "ps/nm";
+        config false;
+        mandatory true;
+        description "chromatic dispersion";
```

Comment:
Capital letter "Chromatic dispersion"

I am also wondering if this description should explicitely indicate that this corresponds to an accumulated value as for PMD ?

Reply:
Correct: the description both for CD and PMD is remained the old one , for triplet ......
Optional penalty associated with a given accumulated
        CD and PMD.
         This list of triplet cd, pmd, penalty can be used to
         sample the function penalty = f(CD, PMD).